accounting/bank/reconciliation/reconciliation_models.rst

```diff
@@ -0,0 +1,122 @@
+=====================
+Reconciliation Models
+=====================
+
+Once the bank statements are correctly imported, it is essential to *reconcile* the records properly
+and ensure all *Journal Entries* are balanced and in agreement. To ease and speed up the
+reconciliation process, you can configure **Reconciliation Models**, which are particularly useful
+with recurrent entries such as bank fees.
+
+.. todo:: Add a link to the Reconciliation process in the paragraph above, once the doc will have
+   been updated.
+
+.. note::
+   Reconciliation Models are also useful to handle *Cash Discounts*. Please refer to
+   :doc:`this documentation <../../receivables/customer_invoices/cash_discounts>` for more
+   information.
+
+.. _reconciliation_models_types:
+
+Types of Reconciliation Models
+==============================
+
+There are three types of Reconciliation Models:
+
+#. :ref:`Write-off Button <reconciliation_models_button>`
+#. :ref:`Suggestion of counterpart values <reconciliation_models_suggestion>`
+#. :ref:`Match existing invoices/bills <reconciliation_models_match>`
+
+.. _reconciliation_models_button:
+
+Manually create a write-off on clicked button
+---------------------------------------------
+
+When you are reconciling an entry with an *Open Balance*, you can use the buttons available under
+the *Manual Operations* tab to pre-fill all the values automatically, before validating the
+reconciliation. Each button is a different Reconciliation Model.
+
+.. image:: media/reconciliation_models_button.png
+   :align: center
+   :alt: Example of a Reconciliation Model with a write-off button in Odoo Accounting
+
+.. _reconciliation_models_suggestion:
+
+Suggest counterpart values
+--------------------------
+
+This type of Reconciliation Model suggests immediately counterpart values that only need to be
+validated. This automation is based on a set of rules defined in the reconciliation model.
+
+.. image:: media/reconciliation_models_suggestion.png
+   :align: center
+   :alt: Example of a Reconciliation Model that suggests counterpart values in Odoo Accounting
+
+.. _reconciliation_models_match:
+
+Match existing invoices/bills
+-----------------------------
+
+This type of Reconciliation Model selects automatically the right Customer Invoice or Vendor Bill
+that matches the payment. All that is left to do is to validate the entry. This automation is based
+on a set of rules defined in the reconciliation model.
+
+.. image:: media/reconciliation_models_match.png
+   :align: center
+   :alt: Example of a Reconciliation Model that matches existing invoices and bills automatically
+         in Odoo Accounting
+
+Configuration
+=============
+
+To manage or create new **Reconciliation Models**, go to :menuselection:`Accounting -->
+Reconciliation --> Reconciliation Models`. Alternatively, you can also open this menu from the
+Accounting Overview, by going to your Bank Journal car, clicking on the three little dots, and then
```

Comment:
```suggestion
Accounting Overview, by going to your Bank Journal **car**, clicking on the three little dots, and then
```

if you meant to use this word: okeeeyy! I couldn't distinguish if it is a typo or a correct way to refer to it ^^

Reply:
@kbr-odoo 
typo! I meant "card". Thank you very much :)